src/wallet/wallet.h

```diff
@@ -1198,6 +1198,11 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
 
     /** Whether a given output is spendable by this wallet */
     bool OutputEligibleForSpending(const COutput& output, const CoinEligibilityFilter& eligibility_filter) const;
+
+    template<typename... Params>
+    void WalletLogPrintf(std::string fmt, Params... parameters) const {
+        LogPrintf(("[" + GetName() + "] " + fmt).c_str(), parameters...);
```

Comment:
This results in empty brackets at the start of the log if the default is being used, eg:

```
2018-06-21T14:12:32.172374Z init message: [] Loading wallet...
2018-06-21T14:12:32.174733Z [] nFileVersion = 169900
2018-06-21T14:12:32.174776Z [] Keys: 0 plaintext, 0 encrypted, 0 w/ metadata, 0 total. Unknown wallet records: 0
2018-06-21T14:12:32.175201Z [] Performing wallet upgrade to 169900
```

Any thoughts about only adding the prefix if the wallet name is non-empty?

Reply:
Agree empty is ugly, I'm ended up changing it to be like the GUI and put `default wallet` there instead, so that it's explicit.